azure-pipelines.yml

```diff
@@ -112,8 +112,16 @@ jobs:
         displayName: 'Test'
 
       - script: |
-          $(Build.SourcesDirectory)\build\$(BuildType)\example.exe"
+          $(Build.SourcesDirectory)\build\$(BuildType)\example.exe
         displayName: 'Example'
+        
+      - script: |
+          copy $(Build.SourcesDirectory)\build\$(BuildType)\FesapiCpp*.dll $(Build.SourcesDirectory)\cs\example\bin\$(BuildType)
```

Comment:
I think that it is better to use task than script when available. Copying a file is a well known task : https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/copy-files?view=azure-devops&tabs=yaml

Reply:
Done.